src/GSMBand.h

```diff
@@ -42,21 +44,34 @@ class GSMBand {
    */
   GSMBand(bool trace = false);
 
-  /** Forces modem hardware restart, so we begin from scratch
-      @return always returns IDLE status
-   */
-  GSM3_NetworkStatus_t begin();
```

Comment:
why begin was moved?

Reply:
i roll back i wrong to past after a test